packages/terra-form-radio/src/RadioField.jsx

```diff
@@ -123,7 +123,7 @@ const RadioField = (props, { intl }) => {
   );
 
   const content = React.Children.map(children, (child) => {
-    if (child.type.isRadio) {
+    if (child && child.type.isRadio) {
```

Comment:
Can you include a Jest test for this scenario?

Reply:
Added: b53561c

Reply:
@tbiethman I tried to run jest to create the snapshot but it gives me an error. What do I need to do to fix this?

```
● Validation Error:

  Module raf/polyfill in the setupFiles option was not found.
         <rootDir> is: /Users/yj048444/git/terra-core

  Configuration Documentation:
  https://jestjs.io/docs/configuration.html
```

Reply:
raf is a dependency of terra-toolkit: https://github.com/cerner/terra-core/blob/master/package.json#L91. Can you verify that those dependencies are present in your node_modules directory?

Reply:
I found the cause. I needed to run compile first LOL
Updated: d95158091